Giveaways.md

```diff
@@ -0,0 +1,15 @@
+# Giveaways
+From time to time Devstaff has the opportunity to perform some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
```

Comment:
Also perform to become `offer`

Reply:
Also, can we amend `Devstaff` to `DevStaff` ?

Reply:
Just updated by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/462/commits/b5477d5ab73b8780e5140cb690798434d8fcf254